subsys/spm/Kconfig

```diff
@@ -61,6 +61,20 @@ config SPM_SERVICE_RNG
 	  This service allows it to request random numbers from the SPM.
 endif
 
+# FIXME: Work-around for Thingy:91 until tooling issues have been worked out.
+config SPM_ALLOW_NON_SECURE_RESET
+	bool "Allow Non-Secure reset"
+	default y if BOARD_NRF9160_PCA20035
+	default n
+	help
+	  This will allow the application running in Non-Secure to do a
+	  system reset of the chip without going through a secure service.
+	  This should be enabled for boards which doesn't have a pin-reset
+	  when doing development/debugging as a debugger working in secure
```

Comment:
Isn't it the other way around, e.g. the debugger running in non-secure can't reset if it's blocked from secure?

Reply:
No see Jira NRF91-313